src/features/metadata/components/EnvBuilds.tsx

```diff
@@ -44,7 +44,13 @@ export const EnvBuilds = ({
           >
             Status: {""}
             <Typography component="span" sx={{ fontSize: "13px" }}>
-              {currentBuild.status}
+              {currentBuild.status_info ? (
+                <>
+                  {currentBuild.status} ({currentBuild.status_info})
+                </>
+              ) : (
+                <>{currentBuild.status}</>
+              )}
```

Comment:
I think you can remove the angle brackets here,

```suggestion
              {currentBuild.status_info ? (
                  {currentBuild.status} ({currentBuild.status_info})
              ) : (
                {currentBuild.status}
              )}
```

Reply:
No, these brackets are required.